man/io_uring_prep_getxattr.3

```diff
@@ -0,0 +1,61 @@
+.\" Copyright (C) 2023 Rutvik Patel <heyrutvik@gmail.com>
+.\"
+.\" SPDX-License-Identifier: LGPL-2.0-or-later
+.\"
+.TH io_uring_prep_getxattr 3 "January 23, 2023" "liburing-2.3" "liburing Manual"
```

Comment:
All of these should go to liburing-2.4 or later.

liburing-2.3 is not being redone.